lib/kanta_web/live/translations/translation_form_live/components/plural_translation_form/plural_translation_form.ex

```diff
@@ -51,7 +52,7 @@ defmodule KantaWeb.Translations.PluralTranslationForm do
         form["nplural_index"] == nplural_index
       end)
 
-    Kanta.Plugins.POWriter.OverwritePoMessage.plural(
+    OverwritePoMessage.plural(
```

Comment:
Inconsistent casing for `PO` name across modules(here "OverwritePoMessage" is used with "Po" instead of "PO").